### PR TITLE
Remove default value from variable name column

### PR DESCRIPTION
**Pull request by mcraveiro**
Seems like a minor typo. Thanks very much for Treemacs!

### CODE REVIEW

File: README.org [hunk 1]
```diff
@@ -514,7 +514,7 @@ Treemacs offers the following configuration options (~describe-variable~ will us
...
 | treemacs-file-name-transformer         | identity                                         | Transformer function that is applied to file names before rendering for any sort of cosmetic effect.                                                                                                                                 |
-| treemacs-user-mode-line-format nil     | nil                                              | When non-nil treemacs will use it as a mode line format (otherwise format provided by ~spaceline~, ~moody-mode-line~ and ~doom-modeline~ will be used or, finally, "Treemacs" text will be displayed)                                    |
+| treemacs-user-mode-line-format         | nil                                              | When non-nil treemacs will use it as a mode line format (otherwise format provided by ~spaceline~, ~moody-mode-line~ and ~doom-modeline~ will be used or, finally, "Treemacs" text will be displayed)                                    |
```
Comment (Alexander-Miller) [inline README.org:517]: Just one nitpick: Readme stuff should go under [Docs].

Comment (mcraveiro) [reply README.org:517]: Hi, sorry this was a bit of a drive-by patch from GitHub just for the typo. Shall I close the PR then? Cheers

Comment (mcraveiro) [reply README.org:517]: Ah, do you mean the subject should be "[Docs]" or is there a Docs folder? Not quite sure I got it.

**Comment by mcraveiro** [comment]
Hm. It seems there is no way to fix a commit from GitHub :-( I'll close the PR and re-do it when I have a sec. Cheers

**Comment by Alexander-Miller** [comment]
Not in github, no, but changing a PR without extra commits is easy enough - just change whatever you need locally and force push.

**Comment by mcraveiro** [comment]
Yeh, I tend to use the GitHub interface for this sort of drive-by PRs. Shame you can't edit the commits from there. Oh well. Thanks for the merge.

**Pull request closed without merging**